src/hooks/useUnitAdministrative.ts

```diff
@@ -0,0 +1,8 @@
+import { ProvinceType } from '../pages/User/types';
+import axiosClient from '../utils/axios/axios';
+
+export const UseUnitAdministrative = async (): Promise<ProvinceType[]> => {
+  return await axiosClient.get(
+    'http://localhost:4000/export-unit-administrative'
+  );
+};
```

Comment:
Đây không phải là cách viết hook
```suggestion
export const useUnitAdministrative = () => {
    const [data, setData] = useState<ProvinceType[]>([]);
    const reFetch = useCallback(async () => {
        const data = await axiosClient.get(
            'http://localhost:4000/export-unit-administrative'
        );
        setData(data);
    }, []);

    useEffect(() => {
        reFetch();
    }, []);

    return { data, reFetch };
};
```